.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change (place an x in the [ ] that applies)
+
+- [ ] New application
+- [ ] Bug fix (non-breaking change which fixes an issue) 
+- [ ] New feature (non-breaking change which adds functionality) 
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
+- [ ] Documentation update
+
+### Summary
+
+Describe the goal of this PR. Mention any related Issue numbers. 
+
+If you are introducing a new sample app in this pull request, please also include a detailed description of:
+
+- The application’s purpose and main functionality
+- What framework(s), languages, and technologies the application uses
+
+### Requirements (place an x in each [ ] that applies)
+
+- [ ] (New apps only) I’ve checked my application against the Checklist to ensure it complies with all standards highlighted there
+- [ ] (New apps only) I have read understand the expectations and responsibilities of submitting a sample application to be made public based on the Terms of Service
+- [ ] I have ensured the changes I am contributing to the app align with the existing patterns of the app and I have tested and linted my code
+- [ ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)
```

Comment:
```suggestion
- [ ] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [ ] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [ ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)
```

ToS doesn't feel like the correct term. I've removed it for the time being until we have a document to reference.